data/registers/rcc_c0.yaml

```diff
@@ -73,10 +80,14 @@ block/RCC:
     description: RCC APB peripheral clock enable in Sleep/Stop mode register 2
     byte_offset: 80
     fieldset: APBSMENR2
-  - name: CCIPR
-    description: RCC peripherals independent clock configuration register
+  - name: CCIPR1
+    description: RCC peripherals independent clock configuration register 1
     byte_offset: 84
-    fieldset: CCIPR
+    fieldset: CCIPR1
```

Comment:
Is the preferred course of action here to leave the name as is, or rename it? Renaming will require a one-line change here:
https://github.com/embassy-rs/embassy/blob/7a97825427b58934329e0ed015d1c201db09251c/embassy-stm32/src/rcc/c0.rs#L190